docs/md/dev-wgs.md

```diff
@@ -50,3 +52,67 @@ The `unmapped_bam_suffix`
 is almost always `.unmapped.bam`,
 so that is its default value
 unless it is specified.
+
+The `reference_fasta_prefix` can be used to override
+the [default value](https://github.com/broadinstitute/wfl/blob/master/api/src/wfl/references.clj#L7) used by this module: 
+"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38"
+
+## Usage
+
+###Create Workload: `/api/v1/create`
+Creates a WFL workload. Before processing, confirm that the WFL and Cromwell service accounts have
+at least read access to the input files.
+```
+curl --location --request POST 'https://dev-wfl.gotc-dev.broadinstitute.org/api/v1/create' \
+--header 'Authorization: Bearer '$(gcloud auth print-access-token) \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+  "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+  "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+  "pipeline": "ExternalWholeGenomeReprocessing",
+  "project": "PO-1234",
```

Comment:
What is project used for in this request? Are we able to query all workloads in a project?

Reply:
This field just a way to help you identify what this workload is for (it is not used by the WGS module to do anything special). So in our tests for example, we set this to the PR branch name.

Reply:
Right now, there is no way to query for workloads based on the project name. Is that something that you need to be able to do?

Reply:
I think that would be a nice-to-have feature. Being able to see everything we've done for a ticket would be useful!

Reply:
Ok good to know! I'll make a ticket for this feature request